src/components/simpleCard/simpleCard.js

```diff
@@ -84,6 +84,7 @@ export default function SimpleCard({ width, image, title, subTitle, detail, pric
     <Card
       style={{
         minWidth: `${width}`,
+        marginRight: "25px",
       }}
       className={classes.root}
```

Comment:
Hey @sakshi-choudhary , instead of giving inline styling you can make use of `className` which is already present there